Gateway/Response/PaymentCaptureDetailsHandler.php

```diff
@@ -41,6 +41,9 @@ public function handle(array $handlingSubject, array $response)
         // set pspReference as lastTransId only!
         $payment->setLastTransId($response['pspReference']);
 
+        if($response["response"]=='[capture-received]'){
+            $payment->setIsTransactionPending(true);
+        }
```

Comment:
I think the lines below should also be included in this if, as it doesn't make sense to close the transaction if the capture request wasn't received. And should we handle an else block to stop the invoice from being created if the response isn't [capture-received] ?